spec/game_spec.rb

```diff
@@ -0,0 +1,339 @@
+require '../lib/game'
+require 'spec_helper'
+require 'pry'
+
+RSpec.describe Game do
```

Comment:
Even though a lot of the test are skipped here, I really like the thought process behind the things you are planning to test. You have a good instinct for testing.